File: C++/polevl.cpp
#include <iostream>

double polevl(double x, double coef[], int N){    
    double ans1;
    ans1 = coef[0];
    
    for (int i = 1; i <= N; i++) {
        ans1 = ans1 * x + coef[i];
    }
    return ans1;
}

File: Methods/Methods-Java/Java source-code/polevl.java


public class polevl {
    public static double polevl_m(Double x, Double[] coef, Integer N)
            throws ArithmeticException {
        double ans;
        ans = coef[0];
        for (int i = 1; i <= N; i++) {
            ans = ans * x + coef[i];
        }
        return ans;
    }
}